utils/CMSApi/remotePublish.ts

```diff
@@ -0,0 +1,46 @@
+import { httpClient, StatusCode } from "configs/api/httpClient/HttpClient";
+import { CMS_API_JWT, CMS_API_VERSION, JWT, STATUS_DRAFT, STATUS_PUBLISH } from "constants/general";
+import { IError } from "interfaces";
+
+const updateJwt = async () => {
+  const result:{data:{token: string}} = await httpClient.post(`${process.env.NEXT_PUBLIC_BASE_URL}${process.env.NEXT_PUBLIC_BASE_ROUTE}${CMS_API_JWT}`, {
```

Comment:
```suggestion
  const { data: { token = '' } = {} } : { data: { token: string } } = await httpClient.post(`${process.env.NEXT_PUBLIC_BASE_URL}${process.env.NEXT_PUBLIC_BASE_ROUTE}${CMS_API_JWT}`, {
```

I guess we need to config linter to put spaces